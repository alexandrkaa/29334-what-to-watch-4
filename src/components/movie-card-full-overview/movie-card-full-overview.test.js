import React from 'react';
import renderer from 'react-test-renderer';
import MovieCardFullOverView from './movie-card-full-overview.jsx';

const movie = {
  id: 4,
  image: `img/we-need-to-talk-about-kevin.jpg`,
  movieBackground: `img/bg-the-grand-budapest-hotel.jpg`,
  movieDate: `2000`,
  movieDescription: [`In the 1930s, the Grand Budapest Hotel is a popula…y boy, becomes Gustave&apos;s friend and protege.`, `Gustave prides himself on providing first-classNam…ess painting and the chief suspect in her murder.`],
  movieDirector: `Wes Anderson`,
  movieGenre: `Thriller`,
  movieImage: `img/the-grand-budapest-hotel-poster.jpg`,
  movieRatingCount: `349 ratings`,
  movieRatingLevel: `Bad`,
  movieRatingScore: `3`,
  movieStarring: `Jude Law, Willem Dafoe, James Franco, Jason Statham, Tom Hardy, Saoirse Ronan, Tony Revoloru, Tilda Swinto`,
  title: `Dardjeeling Limited`,
  moviePreview: `https://upload.wikimedia.org/wikipedia/commons/transcoded/b/b3/Big_Buck_Bunny_Trailer_400p.ogv/Big_Buck_Bunny_Trailer_400p.ogv.360p.webm`,
  backgroundColor: `#000000`,
};

it(`<MovieCardFullOverView /> should movie full card overview to match snapshot`, () => {
  const tree = renderer
    .create(
        <MovieCardFullOverView
          movie={movie}
        />
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
