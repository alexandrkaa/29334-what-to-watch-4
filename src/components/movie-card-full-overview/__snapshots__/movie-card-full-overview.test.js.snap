// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MovieCardFullOverView /> should movie full card overview to match snapshot 1`] = `
Array [
  <div
    className="movie-rating"
  >
    <div
      className="movie-rating__score"
    >
      3
    </div>
    <p
      className="movie-rating__meta"
    >
      <span
        className="movie-rating__level"
      >
        Bad
      </span>
      <span
        className="movie-rating__count"
      >
        349 ratings
      </span>
    </p>
  </div>,
  <div
    className="movie-card__text"
  >
    In the 1930s, the Grand Budapest Hotel is a populaâ€¦y boy, becomes Gustave&apos;s friend and protege.
    <p
      className="movie-card__director"
    >
      <strong>
        Director: 
        Wes Anderson
      </strong>
    </p>
    <p
      className="movie-card__starring"
    >
      <strong>
        Starring: 
        Jude Law, Willem Dafoe
      </strong>
    </p>
  </div>,
]
`;
